tracing/src/main/java/io/airlift/tracing/OpenTelemetryConfig.java

```diff
@@ -21,4 +24,18 @@ public OpenTelemetryConfig setEndpoint(String endpoint)
         this.endpoint = endpoint;
         return this;
     }
+
+    @DecimalMax("1")
```

Comment:
We can use `@Max` here. `@DecimalMax` is only needed for non-integer constraint values.

Note that per the Bean Validation specification, neither validation annotations are required to support `double`, but the Hibernate Validator implementation that we use supports `double` for both annotations.